e2e_samples/temperature_events/Infra/terraform/modules/services/variables.tf

```diff
@@ -80,5 +80,5 @@ variable "retention" {
 variable "set_key_permissions" {
   type        = list(string)
   description = "List of key permissions, must be one or more from the following: backup, create, decrypt, delete, encrypt, get, import, list, purge, recover, restore, sign, unwrapKey, update, verify and wrapKey"
-  default     = ["set", "get", ]
+  default     = ["set", "get", "delete", "purge"]
```

Comment:
who needs the delete and purge permission?

Reply:
The SP that runs the `Terraform` script and CRUDs the keys